src/Card/Card.Superhead.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+import React from "react";
+import { superheadWeights } from "./Card.levers";
+
+export const Superhead = ({ weight, className, children, ...props }) => {
+  const superheadWeight = superheadWeights[weight];
+
+  return (
+    <span
+      className={dcnb("su-type-0 su-mb-0", superheadWeight, className)}
+      {...props}
+    >
+      {children}
+    </span>
+  );
+};
+
+Superhead.propTypes = {
+  /**
+   * Font weight.
+   */
+  weight: PropTypes.oneOf(Object.keys(superheadWeights)),
```

Comment:
FYI adding comments in propTypes using this format will add help text to the Storybook interface, but using // won't.

Reply:
Most interesting. Thanks for sharing.